src/app/features/home/edit-profile/edit-profile.page.ts

```diff
@@ -27,22 +19,21 @@ import { NetworkService } from '../../../shared/network/network.service';
 })
 export class EditProfilePage {
   readonly networkConnected$ = this.networkService.connected$;
-  readonly username$ = this.diaBackendAuthService.username$;
-  readonly avatar$ = this.diaBackendAuthService.avatar$.pipe(
-    shareReplay({ bufferSize: 1, refCount: true })
+  readonly displayName$ = this.diaBackendAuthService.profile$.pipe(
+    map(profile => profile.display_name)
   );
-  readonly profile$ = this.diaBackendAuthService
-    .readProfile$()
-    .pipe(shareReplay({ bufferSize: 1, refCount: true }));
-  readonly description$ = this.profile$.pipe(
+  readonly description$ = this.diaBackendAuthService.profile$.pipe(
     map(profile => profile.description)
   );
-  readonly background$ = this.profile$.pipe(
+  readonly avatar$ = this.diaBackendAuthService.profile$.pipe(
+    map(profile => profile.profile_picture_thumbnail)
+  );
+  readonly background$ = this.diaBackendAuthService.profile$.pipe(
     map(profile => profile.profile_background_thumbnail)
   );
   readonly form = new UntypedFormGroup({});
   model: EditProfileFormModel = {
-    username: '',
+    displayName: '',
```

Comment:
Found it in some cases it referred to as profileName (e.g. transaltion) and sometimes displayName. Maybe use profileName for all Ionic side variables and use `display_name` only on backend API response?